.github/website.sh

```
@@ -21,4 +21,11 @@ git commit --message "Update joda-beans from CI: $GITHUB_ACTION"
 echo "## push..."
 git push origin main
 
+echo "## tidy..."
+cd ..
+git clone https://${GITHUB_TOKEN}@github.com/JodaOrg/joda-beans.git
+cd joda-beans
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)